cypress/integration/admin.js

```diff
@@ -0,0 +1,71 @@
+import LoginPage from '../objects/loginPage';
+import General from '../objects/general';
+import TimeLogging from '../objects/timeLogging';
+import CreateTasks from '../objects/createTasks';
+import CreateClient from '../objects/createClient';
+
+const loginPage = new LoginPage();
+const general = new General();
+const timeLogging = new TimeLogging();
+const createTasks = new CreateTasks();
+const createClient = new CreateClient();
+
+describe('Sourcebook admin actions', function() {
+
+    beforeEach(function() {
+        cy.loginAs('Admin');
+        loginPage.visit();
+    })
+
+    /*it('Validate today\'s date', function () {
+
+        timeLogging.getTodaysDate().contains(new Date().getDate());
+    })
+
+    it('Admin creates new task', function() {
+
+        //Go to tasks page
+        general.getItemFromMenu('tasks').click();
+        general.getRegularButton().click();
+
+        //Create unique task name
+        let taskName = 'TestTask' + new Date().getTime();
+
+        //Fill form fields and submit
+        createTasks.getField('name').type(taskName);
+        createTasks.getField('description').type('Description');
+        createTasks.getBillableField().click();
+        general.getSpecificValueFromDropDown('Yes').click();
+        createTasks.getField('rate').clear().type(2);
+        general.getSubmitButton().click();
+
+        //Check if task was created
+        general.getUrl().should('not.include', 'create');
+        general.getItemFromMenu('tasks').click();
+        general.getFilterFields().first().type(taskName);
+        general.getDisplayedItems().contains(taskName);
+    })*/
+
+    it('Admin creates new client', function() {
+
+        //Go to clients page
+        general.getItemFromMenu('clients').click();
+        general.getRegularButton().click();
+
+        //Create unique organization name
+        let orgName = 'TestOrg' + new Date().getTime();
+
+        //Fill form fields and submit
+        createClient.getField('organization').type(orgName);
```

Comment:
you are still using locators in the test case, this is not a recommended approach from page object model, all locators should be in page object file and not encoded in the test case file